Formula/confluent-platform.rb

```diff
@@ -11,14 +11,36 @@ class ConfluentPlatform < Formula
 
   conflicts_with "kafka", :because => "kafka also ships with identically named Kafka related executables"
 
+  patch :DATA
```

Comment:
Has this been reported upstream? This needs a comment and a link to an upstream issue at a minimum and description of when it should be removed or how it should be updated. In an ideal world, upstream would accept the fix and we would use a `patch do` block to grab the patch from github.